variables.tf

```diff
@@ -4,12 +4,30 @@ variable "create_iam_account" {
   default     = true
 }
 
+variable "create_accessanalyzer" {
+  description = "Controls whether to configure the IAM Access Analyzer"
+  type        = bool
+  default     = false
+}
+
 variable "account_alias" {
   description = "Name of the IAM account alias"
   type        = string
   default     = ""
 }
 
+variable "analyzer_name" {
+  description = "(Required) Name of the Analyzer."
```

Comment:
Not a required variable if you are providing a default

Reply:
Gotcha. The Terraform module requires it, but we won't, due to the default.

Reply:
also updated CHANGELOG.md